src/components/Receiving/ItemDetails.js

```diff
@@ -27,16 +35,21 @@ class ItemDetails extends Component {
   static propTypes = {
     close: PropTypes.func.isRequired,
     linesItemList: PropTypes.object.isRequired,
+    location: PropTypes.object,
     locationsOptions: PropTypes.arrayOf(PropTypes.object).isRequired,
+    mutator: PropTypes.object,
```

Comment:
this is a potential bug, your prop has the same name as stripes-connect provides. You can look into other places in Folio, where mutator is passing below - it gets name parentMutator

Reply:
thanks, good point